xml/ha_glossary.xml

```diff
@@ -10,7 +10,7 @@
  * types of resources: primitives, groups, etc.
  * meta attributes, instance attributes, operations, utilization attributes
  * transition graph
- * daemons: attrd, lrmd, mgmtd, stonithd
+ * daemons: pacemaker-attrd, pacemaker-execd, mgmtd, pacemaker-fenced
```

Comment:
BTW, mgmtd only existed in SLE11.

Reply:
Ah, removed from the comment. That's why I did not find any hits for this term when grepping for it yesterday. ;)